DocumentedExamples/IcePlottingExample.ipynb

```diff
@@ -4,11 +4,9 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.                  "sis": {"expt": "panant-01-zstar-v13",
</pre><p>I would suggest to use the current version which is panat-01-zstar-ACESSyr2 and then you woueld need to change the time period</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>